### PR TITLE
Season

### PR DESCRIPTION
**Pull request by Season0727**

### CODE REVIEW

File: season/string_manipulation.md [hunk 1]
```diff
@@ -19,15 +19,26 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
...
 
 ```ruby
-# write your code here
+  wordsarray = iverson_rant.split
+  first_letters_array = []
+
+  wordsarray.map do |word|
+    first_letters_array.push(word.chr)
+  end 
+
```
Comment (gretchenziegler) [inline season/string_manipulation.md]: Looks good! Two little nitpicks:
* Generally in Ruby, you'll want to separate variable names with underscores (e.g. `words_array` instead of `wordsarray`)
* `.map` is going to make a new array entirely, but on line 25, you don't care about making a new array (since you're pushing to a separate array on line 26). I would use `.each` instead of `.map` in this instance; `.each` will execute some code for each item passed to the block (in this case, `word`), but will return the original array.

File: season/string_manipulation.md [hunk 2]
```diff
@@ -19,15 +19,26 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
...
 ```
 
 2. Programmatically find out exactly how many times Iverson used the word 'practice' in his rant.
 
 ```ruby
-# write your code here
+ iverson_rant.scan("practice").count
 ```
-
+
```
Comment (gretchenziegler) [inline season/string_manipulation.md:41]: Fancy!

File: season/string_manipulation.md [hunk 3]
```diff
@@ -44,5 +55,5 @@ Programmatically do the following to the `sam_i_am` variable:
 Hint: You can chain substitutions!
 
 ```ruby
-# write your code here
+   sam_i_am.gsub(" ","_").gsub("\n","*").gsub("Sam","Season").gsub("o","0").gsub("green","blue")
 ```
```
Comment (gretchenziegler) [inline season/string_manipulation.md:59]: Looks good here 👍

File: season/string_manipulation.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # String Manipulation
 
-## Allen Iverson
+## Allen Allen_Iverson_Practice
```
Comment (gretchenziegler) [inline season/string_manipulation.md:4]: Why did you change the name here? ⛹️

File: season/string_manipulation.md (hunk 2 above)
Comment (Season0727) [reply season/string_manipulation.md:41]: Thank you!

File: season/string_manipulation.md (hunk 4 above)
Comment (Season0727) [reply season/string_manipulation.md:4]: i have no idea how that happened. Sorry.... :(

**Review by gretchenziegler — APPROVED** [review]
